src/storage/src/storage_state.rs

```diff
@@ -379,8 +387,17 @@ impl<'w, A: Allocate> Worker<'w, A> {
         let mut shutdown = false;
         while !shutdown {
             match self.client_rx.recv() {
-                Ok((rx, tx)) => self.run_client(rx, tx),
-                Err(_) => shutdown = true,
+                // We don't use the activator sender here, because we use a `Sequencer` to
```

Comment:
I don't think this comment is correct: we _do_ send back the current thread (which is the timely thread of this worker), because that's what's reading from the command_tx (which then pushes commands into the internal flow).

The code is correct, it seems, though.